ciphey/basemods/Decoders/URL.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional, Dict, List
```

Comment:
Can you please add a test to this? :) 
https://github.com/Ciphey/Ciphey/wiki/Adding-your-own-ciphers#testing-your-code

Reply:
Hey, I added a simple test, I can add some more if needed